src/better_sqlite3.cpp

```diff
@@ -1,2165 +1,2165 @@
-// better_sqlite3.cpp
```

Comment:
What's causing every line in this file to change? Is it a line ending issue?

Seems to have been introduced in 113e8bd567ed83d3bc33c272f3de19465684fb00.

Same for `better_sqlite3.hpp`.

Reply:
No worries, happens to everyone at some point. The `.gitattributes` idea sounds good to me. Also, I didn't know those files were auto generated.